### PR TITLE
feat(api/applications): draft cases

### PR DESCRIPTION
**Pull request by lutamKC**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by francescobrenci-AR — APPROVED** [review]

File: apps/web/src/server/applications/applications.controller.js [hunk 1]
```diff
@@ -20,9 +22,24 @@ import * as applicationsService from './applications.service.js';
 export async function viewDashboard({ params }, res) {
 	const { domainType } = params;
 	const allApplications = await applicationsService.findOpenApplicationsByDomainType(domainType);
-	const readyApplications = allApplications.filter((application) => application.status !== 'draft');
+	const readyApplications = [];
 
-	return res.render('applications/dashboard', { applications: readyApplications });
+	let draftApplications = [];
```
[the remark this answers is not in the document]
Comment (lutamKC) [reply apps/web/src/server/applications/applications.controller.js:27]: no because I re-assign it later to its sorted version

**Pull request merged**